frontend/js/src/year-in-music/2021/YearInMusic.tsx

```diff
@@ -409,24 +412,43 @@ export default class YearInMusic extends React.Component<
             <p>You will find in this page:</p>
             <ul>
               <li>
-                {yourOrUsersName} top <a href="#top-releases">albums</a>,{" "}
-                <a href="#top-recordings">songs</a> and{" "}
-                <a href="#top-artists">artists</a> of the year
+                {yourOrUsersName} top{" "}
+                <a href="listenbrainz/webserver/static/js/src/year-in-music#top-releases">
```

Comment:
These don't look right as HTML links

Reply:
Uh, my IDE must done some automatic refactorings.

Reply:
I figured that was probably what happened. Same in the 2022 file by the way